PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/messages/AzureBundle.java

```diff
@@ -33,11 +36,27 @@ public static String message(@NotNull @PropertyKey(resourceBundle = BUNDLE) Stri
         return ourInstance.getMessage(key, params);
     }
 
+    public static IAzureOperationTitle operation(@NotNull @PropertyKey(resourceBundle = BUNDLE) String name, @NotNull Object... params) {
+        return MessageBundleBasedOperationTitle.builder().name(name).params(params).build();
+    }
+
```

Comment:
I think the naming of "message" is better than "operation".

Reply:
I actually want to distinguish them and use different message bundle so that we can manage them better